client/src/pages/UserAdmin.js

```diff
@@ -0,0 +1,318 @@
+import React, { useState, useEffect } from 'react';
+import '../sass/UserAdmin.scss';
+
+//Parent
+const UserAdmin = () => {
+    
+    const headerToSend = process.env.REACT_APP_CUSTOM_REQUEST_HEADER;
+
+    // Initialize hooks
+    const [users, setUsers] = useState([]); // All users pulled from database
+    const [projects, setProjects] = useState([]); // All projects pulled from db
+    const [userToEdit, setUserToEdit] = useState({}); // The selected user that is being edited
+    const [userManagedProjects, setUserManagedProjects] = useState([]); //  The projects that the selected user is asigned
+    const [searchTerm, setSearchTerm] = useState(""); // Serch term for the user/email search
+    const [projectValue, setProjectValue] = useState("");  // State and handler for form in EditUsers
+    const [userLoaded, setUserLoaded] = useState(false);  // is a user currently loaded
+    const [searchResultType, setSearchResultType] = useState("name"); // Which results will diplay
+
+    // Fetch users from db
+    async function fetchUsers() {
+        try {
+            const res = await fetch("/api/users/", {
+                headers: {
+                    "x-customrequired-header": headerToSend
+                  }
+            });
+            const resJson = await res.json();
+            setUsers(resJson);
+
+        } catch(error) {
+            alert(`fetchUsers ${error}`);
```

Comment:
Don't show the error to a user. Print the error to console instead. Show the following message in the alert:
"Server doesn't respond. Please refresh the page."

Reply:
fixed